src/etc/inc/system.inc

```diff
@@ -1669,7 +1668,7 @@ function system_ntp_setup_gps($serialport) {
 	}
 
 	/* Remove old /etc/remote entry if it exists */
-	if (mwexec("/usr/bin/grep -c '^gps0' /etc/remote")) {
+	if (intval(`/usr/bin/grep -c '^gps0' /etc/remote`)) {
```

Comment:
This should use some form of exec call, we try to avoid using backticks wherever possible.

Reply:
ok, fixed